src/perf_map_poller.rs

```diff
@@ -0,0 +1,152 @@
+use std::{
```

Comment:
Nit: Now that there's two files dealing with maps, they should probably get their own `maps` module.

Reply:
Also I really like that you broke things out this way, removes some bloat from `lib.rs`.

Reply:
For the future, our edition of Rust let's us have the `maps.rs` file side by side with a `maps` directory without a `mod.rs`. `maps.rs` then becomes the root of the module so we don't have lots of files named `mod.rs`. 

I don't think I have a preference on either direction yet, but a thing to know (:

Reply:
Oh nice, probably want to make it part of a larger refactor for consistency then in the future.